test/parallel/test-repl.js

```diff
@@ -224,28 +213,28 @@ function error_test() {
       send: '(function() { "use strict"; return 0755; })()',
       expect: common.engineSpecificMessage({
         v8: /\bSyntaxError: Octal literals are not allowed in strict mode/,
-        chakracore: cc_re4})
+        chakracore: /^SyntaxError: Octal numeric literals and escape characters not allowed in strict mode/}) // eslint-disable-line max-len
```

Comment:
Pulling these regexes out was making them harder to maintain.  Since they have to ignore the length of the lines upstream I didn't see the point in maintaining these exceptions.  In one case the V8 test was failing purely because the regex was formed incorrectly.